specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/managedPrivateEndpoints.json

```diff
@@ -0,0 +1,296 @@
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedVirtualNetworkNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedPrivateEndpointNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "schema": {
+              "$ref": "#/definitions/ManagedPrivateEndpoint"
+            },
+            "description": ""
+          }
+        },
+        "x-ms-examples": {
+          "Get a managed private endpoint": {
+            "$ref": "./examples/GetManagedPrivateEndpoint.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ManagedPrivateEndpoints"
+        ],
+        "operationId": "ManagedPrivateEndpoints_Create",
+        "description": "Create Managed Private Endpoints",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedVirtualNetworkNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedPrivateEndpointNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "schema": {
+              "$ref": "#/definitions/ManagedPrivateEndpoint"
+            },
+            "description": ""
+          }
+        },
+        "x-ms-examples": {
+          "Create a managed private endpoint": {
+            "$ref": "./examples/CreateManagedPrivateEndpoint.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ManagedPrivateEndpoints"
+        ],
+        "operationId": "ManagedPrivateEndpoints_Delete",
+        "description": "Delete Managed Private Endpoints",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedVirtualNetworkNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedPrivateEndpointNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "Already deleted"
+          }
+        },
+        "x-ms-examples": {
+          "Delete a managed private endpoint": {
+            "$ref": "./examples/DeleteManagedPrivateEndpoint.json"
+          }
+        }
+      }
+    },
+    "/managedVirtualNetworks/{managedVirtualNetworkName}/managedPrivateEndpoints": {
+      "get": {
+        "tags": [
+          "ManagedPrivateEndpoints"
+        ],
+        "operationId": "ManagedPrivateEndpoints_List",
```

Comment:
Looks like this returns a pageable response?  If so can you please add [x-ms-pageable](https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-pageable)

Reply:
Added

---
In reply to: [408458563](https://github.com/Azure/azure-rest-api-specs/pull/8753#discussion_r408458563) [](ancestors = 408458563)